avocado/core/xunit.py

```diff
@@ -199,9 +199,7 @@ def end_test(self, state):
             self.xml.add_skip(state)
         elif status == 'FAIL':
             self.xml.add_failure(state)
-        elif status == 'ERROR':
-            self.xml.add_error(state)
-        elif status == 'INTERRUPTED':
+        else:   # ERROR, INTERRUPTED, ...
```

Comment:
This covers the current list of status, but is not generic enough. A new *good* status will call `self.xml.add_error(state)`.
I propose a little change to be prepared for the future:
~~~
if status == 'SKIP':
    self.xml.add_skip(state)
elif mapping[status]:
    self.xml.add_success(state)
elif status == 'FAIL':
    self.xml.add_failure(state)
else:
    self.xml.add_error(state)
~~~